docs/community/team.md

```diff
@@ -71,21 +71,65 @@ Early development on the project was led by Kira Evans, first as an intern with
 
 You can read a full history of the project’s founding and early development in this [blog post](https://ilovesymposia.com/2019/10/24/introducing-napari-a-fast-n-dimensional-image-viewer-in-python/) from Juan.
 
-Over time, napari has grown to over 80 direct contributors. Talley Lambert, from Harvard Medical School, began contributing around the time of Juan's blog post, and has been a critical driver of napari's development ever since. We have core team members located in three continents and three biweekly community calls spread across the day to try to accommodate users and contributors in any time zone.
-
-Read more about napari’s [mission and values](https://napari.org/stable/community/mission_and_values.html), how to get started as a [contributor](https://napari.org/stable/developers/contributing.html) or join us in our [zulip chat](https://napari.zulipchat.com/login/) for a more synchronous conversation. You can also follow us on [mastodon](https://fosstodon.org/@napari).
-
-## Institutional and funding Partners
-
-- [Chan Zuckerberg Initiative Foundation.](https://chanzuckerberg.com/science) Representative [Kyle I. S. Harrington](https://github.com/kephale).
-
-    - The napari project and community have been generously supported by the [CZI Science Imaging Program.](https://chanzuckerberg.com/science/programs-resources/imaging/)
-    - Since late 2018, Juan Nunez-Iglesias has been supported by a CZI Imaging Software Fellowship.
-    - The CZI Imaging Tech Team, under the leadership of Nicholas Sofroniew and engineering management of Justine Larsen, has been making direct engineering and design contributions to improve and expand napari.
-    - The CZI Imaging Tech Team has supported [Quansight Labs](https://labs.quansight.org) and independent contractors to improve and expand napari.
-    - The CZI Imaging Tech Team, through an effort led by Justin Kiggins, has created [napari-hub.org](https://www.napari-hub.org/), a site to make discovering and sharing napari plugins easier.
-    - CZI has launched a [grant program](https://chanzuckerberg.com/rfa/napari-plugin-grants/) to help accelerate the napari plugin developer community.
-
-The current representative of the Institutional and Funding Partner Advisory Council on the napari steering council is Kyle I. S. Harrington.
+Over time, napari has grown to over 180 direct contributors. Talley Lambert,
+from Harvard Medical School, began contributing around the time of Juan's blog
+post, and has been a critical driver of napari's development, including
```

Comment:
```suggestion
post, and was a critical driver of napari's maturation, including
```
Maybe? The current grammatical structure indicates continued involvement by Talley.